damnit/gui/table.py

```diff
@@ -175,6 +178,58 @@ def get_static_columns(self):
 
     def get_static_columns_count(self):
         return self._static_columns_widget.count()
+    
+    def contextMenuEvent(self, event):
+        self.menu = QtWidgets.QMenu(self)
+        self.zulip_action = QtWidgets.QAction('Export table to Zulip', self)
+        self.zulip_action.triggered.connect(self.export_selection_to_zulip)
+
+        self.menu.addAction(self.zulip_action)
+        self.menu.popup(QtGui.QCursor.pos())
+
+    def export_selection_to_zulip(self):
+        if not isinstance(self.model()._main_window.zulip_messenger, ZulipMessenger):
+            self.model()._main_window.zulip_messenger = ZulipMessenger(self.model()._main_window)       
+            
+        selected_rows = [r.row() for r in 
+                         self.selectionModel().selectedRows()]
+        bad_columns = ['Proposal', 'comment_id', 'Status']
+        df = self.model()._main_window.data.iloc[selected_rows]
+        bad_columns = bad_columns + self.columns_with_thumbnails(self.model()._data) \
+            + self.columns_invisible(df)
+        bad_columns = list(dict.fromkeys(bad_columns))
+        df = df.drop(bad_columns, axis=1)
+        if 'Timestamp' in df.columns:
+            df['Timestamp'] = df['Timestamp'].apply(lambda dt: 
+                datetime.fromtimestamp(dt).replace(tzinfo=timezone.utc).\
+                    astimezone().strftime("%H:%M:%S %d/%m/%Y"))
+    
+        msg = df.astype(str).to_markdown(index = False)
+        # Better not to hide legit nan values
+        msg = msg.replace('None', '').replace('<NA>', '')
```

Comment:
you can replace values in the dataframe instead of the string, this is probably less prone to error, see [here](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.replace.html)